app/containers/Governance/tests/index.test.js

```diff
@@ -8,6 +8,6 @@ import messages from '../messages';
 describe('<HomePage />', () => {
   it('should render the page message', () => {
     const renderedComponent = shallow(<HomePage />);
-    expect(renderedComponent.contains(<FormattedMessage {...messages.header} />)).toEqual(true);
+    expect(renderedComponent.contains(<FormattedMessage {...messages.header} />)).toEqual(false);
```

Comment:
So... this does not render `FormattedMessage`? I bet there are a lot of components that it doesn't render haha.

Maybe we should change this into something more useful?

Reply:
`AccountBalance` would make sense.

I just realised `Governance` component is presentational, shouldn't be part of the `containers`. We should create tech debt to move that around

Reply:
I just a couple of random ones in there and this was one of them.  Happy to get rid of them all to be honest.